source/includes/mongodb-compatibility-table-cxx.rst

```diff
@@ -19,7 +19,7 @@
      - MongoDB 2.6
      - MongoDB 2.4
 
-   * - 3.8+
+   * - 3.8 to 3.10
```

Comment:
The rows containing the word "to" are two lines long, which makes the table look a little funny. I experimented with "widths" but couldn't fix it. If you know of a way to fix, let me know!

Reply:
Ah, this is an issue we tried to resolve on some other compat pages to. It's resolved if you reduce the number of overall columns. But that might not be an option here.